src/main/java/com/bitso/BitsoOrderBook.java

```diff
@@ -12,8 +12,8 @@
 public class BitsoOrderBook {
     private Date mOrderDate;
     private int mSequence;
-    private PulicOrder[] mAsks;
-    private PulicOrder[] mBids;
+    private PublicOrder[] mAsks;
+    private PublicOrder[] mBids;
```

Comment:
I can't see any changes in this file. Could you please remove this file of the commit?
Best regards.